exercises/hamming/test/hamming_test.dart

```diff
@@ -5,11 +5,43 @@ void main() {
   final hamming = new Hamming();
 
   group("Hamming", () {
+    test("empty strands", () {
+      final res = hamming.compute("", "");
+
+      expect(res, equals(0));
+    });
+
     test("no difference between identical strands", () {
       final res = hamming.compute("A", "A");
 
       expect(res, equals(0));
-    });
+    }, skip: true);
+
+    test("non-unique character in first strand", () {
+      final res = hamming.compute("AGA", "AGG");
+
+      expect(res, equals(1));
+    }, skip: true);
+
+    test("non-unique character in second strand", () {
+      final res = hamming.compute("AGG", "AGA");
+
+      expect(res, equals(1));
+    }, skip: true);
```

Comment:
I can't see a reasonable difference between the two tests.

For "non-unique character in first strand", if the first strand was "AAG" and the second strand was "AAA", then I would understand the problem.

Then reverse it for "non-unique character in second strand".